CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 3.3.0
+
+* Adapt the web api to make it compatible with `openfisca-core#7`
```

Comment:
I find it confusing to use `#7` since it reminds me issue/PR numbers. Could you use `OpenFisca-Core v7`?